### ISSUE TITLE
is project going strong?

### ISSUE THREAD

**Issue by yarikoptic** [cryo-data/cryo-data#5]
I just saw that https://github.com/cryo-data/data  mentioned https://github.com/cryo-data is not available so decided to ask if you are still pursuing this git-annex/datalad layer of access for the data of interest for you.

**Comment by mankoff** [comment]
Definitely not strong, but still going :). But I am unsure of the long-term viability of this project. It may be too ambitious. Is there something specific you are thinking of with this ticket? Or just general interest in seeing datalad growth to new communities?

**Comment by yarikoptic** [comment]
yeah, more of the latter and because I haven't seen any recent commits and couldn't actually access the subdatasets.

FWIW, re "too ambitious" -- ideally things should be automated as much as possible, but I do understand that it could be hard to impossible to automate fully. nevertheless if data comes from some "reliable" source, then quite feasible -- e.g. look at https://github.com/dandi/dandisets/ superdataset providing access to all those:  https://github.com/dandisets/ .

**Comment by mankoff** [comment]
Yes that DANDI setup is what we are going for in this project.
